components/tools/OmeroWeb/omeroweb/webclient/webclient_gateway.py

```diff
@@ -1453,11 +1452,14 @@ def updatePermissions(self, group, permissions):
 
         perms = str(permissions)
         logger.debug("Chmod of group ID: %s to %s" % (group.id, perms))
-        command = Chmod(type="/ExperimenterGroup",
-                        id=group.id,
-                        permissions=perms)
-        cb = self.c.submit(command)
-        cb.close(True)
+        command = omero.cmd.Chmod(type="/ExperimenterGroup",
+                                  id=group.id,
+                                  permissions=perms)
+        handle = self.c.sf.submit(command, self.SERVICE_OPTS)
```

Comment:
Why are we passing SERVICE_OPTS  and not before? Can we go for a minimal patch at this point? Really the only thing that needs to happen in this whole PR is the addition of `loops=50`.